dist/nearlib.js

```diff
@@ -1252,6 +1256,25 @@ function parseNearAmount(amt) {
     return `${wholePart.add(fractionPart).toString(10, 0)}`;
 }
 exports.parseNearAmount = parseNearAmount;
+/**
+ * Truncates the fraction part of a number to a given number of digits and then trims extra zeroes.
+ * @param value
+ * @param digits
+ */
+function truncateFractionPart(value, digits) {
+    if (digits && value.length > digits) {
+        if (value[digits] >= '5') {
+            return value.substring(0, digits - 1) + String.fromCharCode(value[digits - 1].charCodeAt(0) + 1);
```

Comment:
Looks like test with say `0.999999999999999` is missing.

Reply:
added + fixed a bug